subsys/bluetooth/mesh/lightness_srv.c

```diff
@@ -765,7 +768,8 @@ static int bt_mesh_lightness_srv_start(struct bt_mesh_model *mod)
 	};
 	struct bt_mesh_lightness_set set = { .transition = &transition };
 
-	if (atomic_test_bit(&srv->flags, LIGHTNESS_SRV_FLAG_NO_START)) {
+	if (atomic_test_bit(&srv->flags, LIGHTNESS_SRV_FLAG_NO_START) ||
+	    atomic_test_bit(&srv->flags, LIGHTNESS_SRV_FLAG_NO_PERSISTENT)) {
```

Comment:
I'd argue this shouldn't be blocked by NO_PERSISTENT, we might still want to do startup behavior, now or in the future

Reply:
This fall under the same discussion as the persistent storage scheme, and should be discussed in that context.